File: typescript-client-sdk/src/sdk/models/shared/noteupdate.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// NoteUpdate
/** 
 * Update an existing Note. Please note that updating associations is not currently supported.
**/
export class NoteUpdate extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=additional" })
  additional?: Record<string, any>;

  @SpeakeasyMetadata({ data: "json, name=content" })
  content?: string;

  @SpeakeasyMetadata({ data: "json, name=userId" })
  userId?: string;
}

File: go-client-sdk/pkg/models/shared/noteupdate.go
package shared

// NoteUpdate
// Update an existing Note. Please note that updating associations is not currently supported.
type NoteUpdate struct {
	Additional map[string]interface{} `json:"additional,omitempty"`
	Content    *string                `json:"content,omitempty"`
	UserID     *string                `json:"userId,omitempty"`
}
